src/splitInteger.test.js

```diff
@@ -8,12 +8,42 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
+    const result = splitInteger(1, 1);
 
+    expect(result).toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
-
+    expect(splitInteger(1, 1)).toEqual([1]);
   });
-  
+
   // write more tests here
+  it(`should work with negativ 'num'`, () => {
+    expect(splitInteger(-4, 2)).toEqual([-2, -2]);
+  });
```

Comment:
'The input to your function will always be valid', so you need to test only positive integers